docs/index.md

```diff
@@ -11,18 +11,103 @@ on the files.
+        on the position of the sgRNA in the amplicon sequence to trim the reads. </li>
+        <li><b>--exclude_bp_from_right</b>: This argument can be used to trim reads that have low sequencing quality at 
+        at the ends.</li>
+    </ul>
+</ol>
+<p><b>Sample batch file shown below:</b></p>
+<table>
+    <tbody>
+        <tr>
+            <th><b>name</b></th>
+            <th><b>fastq_r1</b></th>
+            <th><b>amplicon_seq</b></th>
+            <th><b>guide_seq</b></th>
+            <th><b>w</b></th>
+            <th><b>wc</b></th>
+        </tr>
+        <tr>
+            <td>BEV_010_F1_A1_R1_A1</td>
+            <td>validation-inputs/BEV/Plate1/BEV_010_F1_A1_R1_A1.construct.fastq.gz</td>
+            <td>GCTATTTAGTGTTATCCAAGGAACATCTTCAGTATCTCTAGGATTCTCTGAGCATGGCAGTTTCTGCTTAT</td>
+            <td>GGAACATCTTCAGTATCTCT</td>
+            <td>20</td>
+            <td>-10</td>
+        </tr>
+        <tr>
+            <td>BEV_016_F1_A2_R1_A2</td>
+            <td>validation-inputs/BEV/Plate1/BEV_016_F1_A2_R1_A2.construct.fastq.gz</td>
+            <td>TTATATACCTTTTGGTTATATCATTCTTACATAAAGGACACTGTGAAGGCCCTTTCTTCTGGTTGAGAA</td>
+            <td>GTTATATCATTCTTACATAA</td>
+            <td>1</td>
+            <td>-3</td>
+        </tr>
+    </tbody>
+</table>
+<p>The sample "BEV_010_F1_A1_R1_A1" is a base editor sample and "BEV_016_F1_A2_R1_A2" is an SpCas9 knockout sample as 
+indicated by the "w" and "wc" columns.</p>
 
+<p>
+<b>Step 5: Running CRISPResso2</b><br>To run CRISPResso2,
+<ol>
+    <li>Make sure Docker is running on your computer. If it is, you should see the Docker logo on your menu bar which 
+    when clicked on should say "Docker Desktop is running".</li> 
+    <li>Make sure you are running CRISPResso2 in the folder with the batch file. This can be done using the cd command.
+    The outputs will be generated in this folder as well. </li>
+    <li>Finally, open your terminal and type the following command:<br>
+    <b>docker run -v ${PWD}:/DATA -w /DATA -i pinellolab/crispresso2 CRISPRessoBatch --batch_settings [batch file name] 
+    --skip_failed --suppress-plots</b></li>
+</ol>
 
+<p>
+<b>Few additional tips/tricks:</b>
+<ul>
+    <li>For experiments with both knockout and base editor samples, the quantification window size and window center 
+    parameters can be toggled appropriately to run all samples together.</li>
+    <li>--skip_failed is a good argument to use while running batch mode. This argument makes sure the job does not quit
+    if one sample fails. The failed sample can be identified in the RUNNING_LOG file. </li>
+    <li>One common error you may encounter is <b>"CRISPResso batch #x was killed by your system. Please decrease the 
+    number of processes (-p) and run again."</b> Using  the --suppress-plots or --suppress-report arguments might fix this 
+    error.  If not, please try increasing the number of resources such as "CPUs" and "Memory" in your  Docker 
```

Comment:
For a first time user, it might be a bit daunting to mess around with the CPUs and Memory preferences in Docker, so maybe give an example parameter change? I was also thinking of adding an email they can contact in case they run into issues, but we also don't want people contacting you with every little CRISPResso issue, so that's not ideal